core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -626,9 +626,12 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
   def listManagedGroupsForUser(): Future[PerRequestMessage] = {
     dataSource.inTransaction { dataAccess =>
-      dataAccess.managedGroupQuery.listManagedGroupsForUser(RawlsUserRef(RawlsUserSubjectId(userInfo.userSubjectId))).map { groupsWithAccess =>
+      for {
+        groupsWithAccess <- dataAccess.managedGroupQuery.listManagedGroupsForUser(RawlsUserRef(RawlsUserSubjectId(userInfo.userSubjectId)))
+        emailsByGroup <- dataAccess.rawlsGroupQuery.loadEmails(groupsWithAccess.map(_.managedGroupRef.toMembersGroupRef).toSeq)
```

Comment:
I looked for a way to do this without an additional query but the change is too deep... another query should be cheap.